cli/config.go

```diff
@@ -0,0 +1,82 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+	"github.com/pelletier/go-toml"
+)
+
+type Config struct {
+	AuthToken string `toml:"auth_token"`
+}
+
+// Save - store config in a file
+func Save(c Config, file string) error {
+	b, err := toml.Marshal(c)
+	if err != nil {
+		return errors.New(fmt.Sprintf("Error reading config file: %s", err))
```

Comment:
Please, avoid using `error` in error messages.  Maybe you can use `failed to read config file`